### PR TITLE
add read the docs yaml

### PR DESCRIPTION
**Pull request by lfunderburk**
Adding read the docs yaml file

solves #1 

@edublancas want to verify this is correct before adding to the main branch

Submitted the read-the-docs.yaml file

### CODE REVIEW

File: readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    pre_build:
+      - aws configure set aws_access_key_id ${aws_access_key_id}
+      - aws configure set aws_secret_access_key ${aws_secret_access_key}
+    # upload to S3
+    post_build:
+      - conda env export --no-build > environment.lock.yml
+      - cat environment.lock.yml
+
+conda:
+  environment: doc/environment.lock.yml
```
Comment (edublancas) [inline readthedocs.yml:19]: you're missing this file

**Review by edublancas — CHANGES_REQUESTED** [review]

File: readthedocs.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  jobs:
+    # download latest version from S3 to leverage notebook cache
+    pre_build:
+      - aws configure set aws_access_key_id ${aws_access_key_id}
+      - aws configure set aws_secret_access_key ${aws_secret_access_key}
+    # upload to S3
+    post_build:
+      - conda env export --no-build > environment.lock.yml
+      - cat environment.lock.yml
```
[the remark this answers is not in the document]
Comment (lfunderburk) [reply readthedocs.yml]: removed

File: readthedocs.yml (hunk 1 above)
Comment (lfunderburk) [reply readthedocs.yml]: resolved in latest commit

**Review by edublancas — APPROVED** [review]
if you have issues with the lock file (I've had issues in the past), you can generate a regular environment.yml (with no pinned dependencies)

**Pull request merged**